cypress/e2e/utils.ts

```diff
@@ -13,9 +13,7 @@ export function shouldBeVisible($subject: Parameters<(typeof cy)['wrap']>[0]) {
   }
 
   cy.get('@subject').should('not.have.attr', 'aria-hidden')
-  cy.get('@subject')
-    .find('.dialog-content', { log: false })
```

Comment:
Note to myself: this line caused Cypress to encouter a weird error in the reworked web component tests where it would incorrectly think the dialog content is fixed and obscured. I couldn’t find a fix so I removed it.